conditions.go

```diff
@@ -166,6 +167,13 @@ func (a *Address) UnmarshalJSON(raw []byte) error {
 		}
 		*a = c.Address()
 		return nil
+	case "bech32":
+		_, addr, err := bech32.Decode(enc)
+		if err != nil {
+			return errors.Wrapf(err, "deserialize bech32: %s", err)
+		}
+		*a = addr
+		return nil
```

Comment:
Hmmm... after seeing the test with invalid address... I would suggest adding `a.Validate()` somewhere in this code path... and in the "hex" codepath as well. If the Condition.deserialize doesn't error, then `c.Address()` is guaranteed to be 20 bytes.

Reply:
Done in https://github.com/iov-one/weave/pull/415/commits/8c2c36251bf16de923a06da8fddd673c1923b4b9